CMakeLists.txt

```diff
@@ -573,8 +576,10 @@ endif(${CMAKE_SYSTEM_NAME} MATCHES "Darwin" AND NOT BUILD_WITH_CONDA)
     ENDIF(NOT PYTHONLIBS_FOUND)
 
 # -------------------------------- pcl ----------------------------------
-    #PCL needs to be found before boost because the PCLConfig also calls find_package(Boost ...),
-    #but with different components
+# Can be used by ReverseEngineering module"
```

Comment:
It is also used by certain 3rd Party Addons
For example @microelly2 NURBS https://github.com/microelly2/freecad-nurbs